### PR TITLE
apr-util: new package

### LINKED ISSUE #12543: [Wolfi Package Request]: apr-util
### What software would you like us to add to wolfi-os. Ideally include a URL to the project and its source.

Apache Portable Runtime Utility: https://apr.apache.org/
Download source: https://apr.apache.org/download.cgi
Also available as Alpine package apr-util.

### which versions of the software should we include?

Latest

### Add some justification for why this specific package and versions are important.

This is a required dependency of subversion: #12486

### Please check all that apply

- [X] This package has an un-restrictive license
- [X] The package/versions proposed are actively maintained upstream
- [X] I am interested in adding this package to Wolfi OS myself
- [ ] I am willing to help maintain this package

### PR DESCRIPTION
**Pull request by bgermann**
Fixes: #12543

#### For new package PRs only

- [x] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For version bump PRs

- [x] The `epoch` field is reset to 0

### CODE REVIEW

File: apr-util.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package:
+  name: apr-util
+  version: 1.6.3
+  epoch: 0
+  description: The Apache Portable Runtime Utility Library
+  copyright:
+    # Some files also contain RSA-MD.
+    # See https://www.ietf.org/ietf-ftp/ietf/IPR/RSA-MD-all
+    # for details on using it without license, i.e. under Apache-2.0 in this case.
+    - license: Apache-2.0 AND Public-Domain AND CC0-1.0 AND OLDAP-2.7 AND Beerware
```
Comment (ajayk) [inline apr-util.yaml:10]:
```suggestion
    - license: Apache-2.0
```

Comment (bgermann) [reply apr-util.yaml:10]: Shouldn't the license list be complete?

Comment (ajayk) [reply apr-util.yaml:10]: we use the license of the source package similar to arch linux and alpine , the LICENSE file in the apr-utils refers as Apache-2.0 

 @imjasonh , @kaniini  : for further clarification

Comment (ajayk) [reply apr-util.yaml]: I had a chat with @imjasonh , list of license you have seems to be good

**Review by ajayk — APPROVED** [review]

**Pull request merged**